src/Features/ERDDAP/hooks/TableDAPComponents.tsx

```diff
@@ -157,10 +158,11 @@ export const UseDataset: React.FunctionComponent<UseDatasetProps> = ({
   children,
   timeSeries,
   startTime,
+  endTime,
   loading,
   error,
 }) => {
-  const { isLoading, data } = useDataset(timeSeries, startTime)
+  const { isLoading, data } = useDataset(timeSeries, startTime, endTime)
 
   if (isLoading) {
     if (loading) {
```

Comment:
I'll look into this!

Reply:
Okay, so we weren't actually showing `PlatformLoadingAlert` on non-wind observations (i.e. fetching one dataset using `useDataset`) so that should be fixed now. 
I'm not sure why wind is not working locally on your end as it is set up to use `useDatasets` and the fallback for loading is showing the loading alert. I can see it always locally. I'm wondering if you aren't seeing the loading components because its loading quickly? If you adjusted the throttling in the dev tool network tab, are you able to reproduce what you were seeing previously (no loading components)? If its still not working, I can dig deeper but from what I can see, wind should be showing loading components for all three of the datasets its loading!

Here's it on slow 4g:
https://github.com/user-attachments/assets/171c3598-340b-4dfa-83d4-2838cd6a1c87